src/components/ElvaYBPractice2/MainComponent.tsx

```diff
@@ -0,0 +1,130 @@
+            <PageViewContainer>{1}M PAGEVIEWS</PageViewContainer>
+          )
+        ) : views === 0 ? (
+          <PageViewContainer>{10}k PAGEVIEWS</PageViewContainer>
+        ) : views === 25 ? (
+          <PageViewContainer>{50}k PAGEVIEWS</PageViewContainer>
+        ) : views === 50 ? (
+          <PageViewContainer>{100}k PAGEVIEWS</PageViewContainer>
+        ) : views === 75 ? (
+          <PageViewContainer>{500}k PAGEVIEWS</PageViewContainer>
+        ) : (
+          <PageViewContainer>{1}M PAGEVIEWS</PageViewContainer>
+        )}
+        <PriceMonthContainer>
+          {toggle === false ? (
+            views === 0 ? (
+              <PriceContainer>$8.00</PriceContainer>
+            ) : views === 25 ? (
+              <PriceContainer>$12.00</PriceContainer>
+            ) : views === 50 ? (
+              <PriceContainer>$16.00</PriceContainer>
+            ) : views === 75 ? (
+              <PriceContainer>$24.00</PriceContainer>
+            ) : (
+              <PriceContainer>$36.00</PriceContainer>
+            )
+          ) : views === 0 ? (
+            <PriceContainer>$6.00</PriceContainer>
+          ) : views === 25 ? (
+            <PriceContainer>$9.00</PriceContainer>
+          ) : views === 50 ? (
+            <PriceContainer>$12.00</PriceContainer>
+          ) : views === 75 ? (
+            <PriceContainer>$18.00</PriceContainer>
+          ) : (
+            <PriceContainer>$27.00</PriceContainer>
+          )}
+          <MonthContainer>/month</MonthContainer>
+        </PriceMonthContainer>
+      </PageViewPriceContainer>
+      <Slidercontainer>
+        <StyleSlider
+          marks={marks}
+          onChange={(value: React.SetStateAction<number>) => setViews(value)}
+          step={null}
+          defaultValue={50}
+          handleStyle={{ backgroundColor: 'hsl(174, 86%, 45%)' }}
+          trackStyle={{ backgroundColor: 'hsl(174, 86%, 45%)' }}
+        />
+      </Slidercontainer>
+      <ToggleContainer>
+        <ToggleContainerText>Monthly Billing</ToggleContainerText>
+        <Switch onChange={(checked) => setToggle(checked)} />
+        <ToggleContainerText>Yearly Billing</ToggleContainerText>
+        <DiscountButton1>25% discount</DiscountButton1>
+        <DiscountButton2>25%</DiscountButton2>
+      </ToggleContainer>
+      <MainBottomComponent>
+        <BottomLeftContainer>
+          <BottomLeftText1>
+            <CheckIcon src={IconCheck} alt="Tick"></CheckIcon>Unlimited websites
+          </BottomLeftText1>
+          <BottomLeftText2>
+            <CheckIcon src={IconCheck} alt="Tick"></CheckIcon>100% data ownership
+          </BottomLeftText2>
+          <BottomLeftText3>
+            <CheckIcon src={IconCheck} alt="Tick"></CheckIcon>Email reports
+          </BottomLeftText3>
+        </BottomLeftContainer>
+        <TrialButtonContainer>
+          <StartTrialButton>Start my trial</StartTrialButton>
+        </TrialButtonContainer>
+      </MainBottomComponent>
```

Comment:
consider extracting the whole Botton Component. You can even make it a component and pass in props for the texts and the IconCheck